public/index.html

```diff
@@ -26,7 +26,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>SNEK</title>
```

Comment:
Consider improving the title. `SNEK` in the browser tab looks a bit minimalistic for my taste.

```suggestion
    <title>SNEK - Social Network for Engineers and Knowledged</title>
```

Reply:
> Consider improving the title. `SNEK` in the browser tab looks a bit minimalistic for my taste.

Isn't this a bit too long for a tab?

Reply:
I think `SNEK Network` would fit.
```suggestion
    <title>SNEK Network</title>
```